test/compress/arrays.js

```diff
@@ -8,7 +8,7 @@ holes_and_undefined: {
     expect: {
         w=[1,,];
         x=[1,2,void 0];
-        y=[1,,2];
+        y=[1,,2,];
```

Comment:
A trailing comma is not necessary in this case:
```
$ node -p '[1, , 2, ].length'
3

$ node -p '[1, , 2].length'
3
```
A trailing comma is only required when the last element is an Array hole:
```
$ node -p '[1,,,].length'
3

$ node -p '[1,,].length'
2
```